plugins/destination/mssql/queries/templates/create_table.sql.tpl

```diff
@@ -1,8 +1,10 @@
 CREATE TABLE {{.Table}} (
 {{with .Definitions}}{{template "col_defs.sql.tpl" .}}{{end}}
 {{- with .PrimaryKey}}
-  CONSTRAINT {{.Name}} PRIMARY KEY (
-{{template "col_names.sql.tpl" .Columns}}
+  {{- if ne (len .Columns) 0}}
```

Comment:
you can just use `{{with}}` as for slices of len = 0 (nil, too) it doesn't do anything